src/PointerEngine.ts

```diff
@@ -106,8 +106,15 @@ export class PointerEngine extends BaseObserver<PointerEngineListener> {
   refreshDeviceList() {
     this.logger.info('Refreshing device list');
     let pointerDevices = _.filter(devices(), (d) => d.usage === 2 && d.usagePage === 1);
-    let unique = _.uniqBy(pointerDevices, (u) => u.serialNumber);
-
+    const uniqueByProduct = _.uniqBy(
```

Comment:
just a small nit while looking at this, wondering if this could be: `_.uniqBy(pointerDevices, (u) => u.serialNumber || u.product );` thoughts?

Reply:
Good catch 
Yeah that would simplify the logic nicely
I tested it and it works great :)